tests/test_lights.py

```diff
@@ -0,0 +1,219 @@
+import math
+
+import pytest
+import vtk
+
+import pyvista
+from pyvista.plotting import system_supports_plotting
+# TODO: do we need OFF_SCREEN stuff? And as per tests/test_renderer.py or tests/plotting/test_plotting.py?
+# TODO: do we actually need the @skip_no_plotting decorator when there's no plotter?
```

Comment:
We do not need the decorator.  We can create lights and VTK/pyvista objects all we want without a framebuffer, it's rendering it that causes a segfault.
```suggestion
```